Sources/ActorSingletonPlugin/ActorSingletonProxy.swift

```diff
@@ -12,239 +12,387 @@
 //
 //===----------------------------------------------------------------------===//
```

Comment:
Funny filename actually -- the uppercase `I` there hah

Reply:
Hmm??  o_O